### DISCUSSION TITLE
How can I contribute to the RBA GitHub?

### DISCUSSION THREAD

**Discussion by ZachTheSplunker** [splunk/rba#17] [Q&A] [answered]
Hello!

What are some ways community members contribute to the RBA GitHub?

**Comment by ZachChristensen28** [comment, marked as answer]
There are many ways to contribute. I would start by looking at the contributing guidelines here: https://splunk.github.io/rba/contributing/contributing-guidelines/.

Some of the ways to contribute are:

- Create a pull request with your proposed changes.
- Create a discussion/idea for something you would like to see.
- Post to our RBA community slack channel -> https://outpost-security.com/slack.
- Open an Issue/feature request on GitHub.
- Reach out to the RBA team directly.
